### DISCUSSION TITLE
软件安装不成功

### DISCUSSION THREAD

**Discussion by DecoLucp** [Molunerfinn/PicGo#612] [Q&A]
我之前安装过PicGo ，我想安装新版本的时候删除了源文件，导致安装不成功
在程序里还有显示，但不能卸载，需要提供 Uninstall PicGo.exe文件 
想询问一下这种问题该怎么解决，麻烦您了

**Comment by Molunerfinn** [comment]
这个应该是windows软件卸载的通用问题，百度一下吧..

**Comment by DecoLucp** [comment]
好的
在2021年1月23日 21:55，PiEgg<notifications@github.com> 写道：

这个应该是windows软件卸载的通用问题，百度一下吧..
